src/service/pull_request_query.ts

```diff
@@ -0,0 +1,123 @@
+import { graphql, GraphqlResponseError } from '@octokit/graphql';
+import { GET_REPO_DATA, IS_ARCHIVED_REPO } from './graphql/pull_request';
+import { RequestParameters } from '@octokit/types'
+import { Logger } from 'winston';
+
+interface Comment {
+    author: {
+        login: string;
+    }
+    body: string; 
+    createdAt: string; 
+    updatedAt: string
+}
+
+
+interface Review {
+    author: {
+        login: string;
+    };
+    comments: Comment[];
+    body: string;
+    state: string;
+    createdAt: string;
+    updatedAt: string; 
+}
+
+interface Label {
+    color: string; 
+    name: string;
+}
+
+// PullRequest
+interface PullRequest {
+    nodes: Label[]; 
+    number: number;
+    title: string;
+    state: string;
+    body: string; 
+    url: string;
+    stateDuration: number; 
+    createdAt: string;
+    updatedAt: string;
+    author: {
+        login: string;
+    };
+    reviews: {
+        nodes: Review[];
+        
+    };
+}
+
+// Set the interface for the expected response regarding Pull Requests
+interface PullRequestsData {
+    repository: {
+        pullRequests: {
+            nodes: PullRequest[];
+        };
+    };
+}
+
+interface RepoCheck {
+    repository: {
+        isArchived: boolean; 
+    }
+}
+
+export interface graphqlInput extends RequestParameters {
```

Comment:
You should capitalize interfaces
```suggestion
export interface GraphqlInput extends RequestParameters {
```

Reply:
fixed it